sklearn/neighbors/tests/test_neighbors.py

```diff
@@ -916,6 +916,15 @@ def test_neighbors_badargs():
                   nbrs.radius_neighbors_graph,
                   X, mode='blah')
 
+    nbrs = neighbors.NearestNeighbors().fit(X)
```

Comment:
What I meant was "is this duplication of lines introduced by a failed merge".
These exact lines are above already, right?